python/graphstorm/gsf.py

```diff
@@ -1144,3 +1149,62 @@ def create_evaluator(task_info):
             config.early_stop_rounds,
             config.early_stop_strategy)
     return None
+
+def create_lp_evaluator(config):
+    """ Create LP specific evaluator.
+
+        Parameters
+        ----------
+        config: GSConfig
+            Configuration.
+    """
+    assert all((x.startswith(SUPPORTED_HIT_AT_METRICS) or x == 'mrr') for x in
+               config.eval_metric), (
+        "Invalid LP evaluation metrics. "
+        "GraphStorm only supports MRR and Hit@K metrics for link prediction.")
+
+    if config.report_eval_per_type:
+        if len(config.eval_metric) == 1 and config.eval_metric[0] == 'mrr':
+            return GSgnnPerEtypeMrrLPEvaluator(eval_frequency=config.eval_frequency,
```

Comment:
Can we use GSgnnPerEtypeLPEvaluator instead?

Reply:
Yes, GSgnnPerEtypeMrrLPEvaluator and GSgnnPerEtypeHitsLPEvaluator deprecated.